### DISCUSSION TITLE
Thank you for this project ❤️

### DISCUSSION THREAD

**Discussion by aviwad** [tranxuanthang/lrclib#25] [General]
Have sent $5 via GitHub donations. This is great. We need a user contributed lyrics website. I will be using this as an alternate source for when Spotify does not have the lyrics in my app (https://github.com/aviwad/LyricFever). Hopefully I can also try uploading the Spotify lyrics to LRCLIB if it does not exist in LRCLIB. In the future. If you believe that to be useful. Thanks a lot.

**Comment by tranxuanthang** [comment]
Thanks a ton for the support! Really appreciate it. Feel free to upload any missing lyrics to LRCLIB, I would love to see it grow with user contributions. Let me know if you run into anything or have more ideas!
